templates/blog/post_detail.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
```

Comment:
The `{% load crispy_forms_filters %}` tag is used to load crispy forms filters, which is correct for using crispy forms in this template. Ensure that the crispy forms package is installed and configured properly in your Django settings.